packages/sdk/package.json

```diff
@@ -24,7 +24,7 @@
     "clean": "rimraf dist dist-* temp types *.tgz *.log",
     "format": "prettier --write --config ./.prettierrc.json --ignore-path ./.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser-case": "karma start --single-run --glob=./dist-test/index.integration.browser.js",
-    "integration-test:browser": "node test/integration/scripts/DownloadSimpleAuth.js && concurrently --kill-others \"node test/integration/scripts/StartSimpleAuth.js\" \"npm run integration-test:browser-case\" || true",
+    "integration-test:browser": "node test/integration/scripts/DownloadSimpleAuth.js && concurrently --kill-others --success first \"node test/integration/scripts/StartSimpleAuth.js\" \"npm run integration-test:browser-case\"",
```

Comment:
should it be `--success all`?

Reply:
the default config is all, but there will be some error when killing the dotnet process
https://github.com/kimmobrunfeldt/concurrently/issues/127
if set to first, there will ignores exit code 1 exiting dotnet process.